app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryUpdate.kt

```diff
@@ -75,6 +75,15 @@ class BloodPressureEntryUpdate(
         }
       }
 
+      is SaveClicked -> {
```

Comment:
Same as earlier comment to collapse `result` into the subject of a `when` block.

Reply:
Same as above, when it comes to Mobius classes.